shippable.yml

```diff
@@ -0,0 +1,64 @@
+branches:
+    only:
+       - mda_arm_ci_shippable
+
+language: python
+
+python:
+    # NOTE: Docker options in pre_ci_boot section 
+    # are not allowed on restricted shared node pools.
+    # so haven't had luck getting Python 3.6 working
+    # on ARM; try 2.7 for now
+    - 2.7
+    # NOTE: some pretty major issues with our test infrastructure
+    # / dependencies on 3.7 it seems, so hold off on that for now
+
+runtime:
+    # use the free open source pool of nodes
+    # only for ARM platform
+    nodePool: shippable_shared_aarch64
+
+build:
+    ci:
+    # install dependencies
+    - sudo apt-get install libfreetype6-dev pkg-config libpng12-dev build-essential
+    - sudo apt-get install gcc gfortran libblas-dev liblapack-dev
+
+    - pip install --upgrade pip pytest
+    - pip install cython --cache-dir=/root/.cache/pip/wheels/$SHIPPABLE_PYTHON_VERSION
```

Comment:
I suspect we may be able to remove this cache path code duplication, but messing with the cache is time-consuming if you accidentally reset it!